WISE-Community/WISE
# Removed message to write down username

...if new student account was created with google.

How to test:
1. Create new student account using Google.
2. In the confirmation page, verify that you don't see the text "Please write this down. You will need it when signing in to WISE in the future."

Closes #1682
